evergreen/set-up-env.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This script runs in a container built from the Dockerfiles.
+# It assumes the workspace directory is mounted on /etc/envoy-serverless and the build directory is optionally mounted on /build
+
+set -xeuo pipefail
+
+export bazel_flags="--config=linux --config=clang -c opt --action_env=PATH --verbose_failures -k"
+
+if grep -q "CentOS Linux 7" /etc/os-release; then
```

Comment:
I tested the OS this way because this script runs in the container. Alternatively, I could pass the environment variable `$DISTRO` from the build variant -> evergreen function -> docker -> this script. I felt this way is simpler.